megamek/src/megamek/client/bot/princess/BehaviorSettingsFactory.java

```diff
@@ -229,7 +244,7 @@ public boolean saveBehaviorSettings(boolean includeTargets) {
         init(false);
         List<String> names;
         synchronized (behaviorMap) {
-            names = new ArrayList<String>(behaviorMap.keySet());
+            names = new ArrayList<>(behaviorMap.keySet());

```

Comment:
This is actually the preferred way of using generics since JDK 7 - the "diamond operator" reduces the amount of code you have to type by using the type definition from the declaring variable.